Examples/Algorithms/HepMC/include/ActsExamples/HepMC/HepMCProcessExtractor.hpp

```diff
@@ -50,7 +52,7 @@ class HepMCProcessExtractor final : public ActsExamples::BareAlgorithm {
   };
 
   /// Constructor
-  HepMCProcessExtractor(Config&& cnf, Acts::Logging::Level level);
+  HepMCProcessExtractor(Config cfg, Acts::Logging::Level level);
```

Comment:
Any reason for the change here?

Reply:
To simplify usage: `Config` contains only values and there is no need to transfer ownership here. Could be `const Config&` but I am keeping a copy internally anyways.

Reply:
Ok